mod_reforged/config/z_nested_tooltips.nut

```diff
@@ -91,8 +91,10 @@ local getThresholdForInjury = function( _script )
-		InjuryThreshold = ::MSU.Class.BasicTooltip("Injury Threshold", ::Reforged.Mod.Tooltips.parseString("If the damage received to [Hitpoints|Concept.Hitpoints] is at least " + ::MSU.Text.colorNegative(::Const.Combat.InjuryMinDamage) + " and is greater than a certain percentage of the current [Hitpoints,|Concept.Hitpoints] the character receives an [injury.|Concept.InjuryTemporary] This percentage can be modified by certain [perks|Concept.Perk] or traits of both the attacker and the target.\n\nCertain [injuries|Concept.InjuryTemporary] require this percentage to be at least a certain value before they can be inflicted, with heavier [injuries|Concept.InjuryTemporary] requiring a higher percentage.\n\nFor example the threshold for [Cut Arm|Skill+cut_arm_injury] is " + ::MSU.Text.colorNegative(getThresholdForInjury("injury/cut_arm_injury") + "%") + " and that of [Split Hand|Skill+split_hand_injury] is " + ::MSU.Text.colorNegative(getThresholdForInjury("injury/split_hand_injury") + "%") + ".")),
-		Reach = ::MSU.Class.BasicTooltip("Reach", ::Reforged.Mod.Tooltips.parseString("Reach is a depiction of how far a character\'s attacks can reach, making melee combat easier against targets with shorter reach.\n\n[Melee skill|Concept.MeleeSkill] is increased when attacking opponents with shorter reach, and reduced against opponents with longer reach. Reach has diminishing returns, starting at " + ::Reforged.Reach.BonusPerReach + " and dropping by 1 to a minimum of 1. It only applies when attacking a target adjacent to you or up to 2 tiles away with nothing between you and the target.\n\nAfter a successful hit, the target\'s [Reach Advantage|Concept.ReachAdvantage] is lost until the attacker waits or ends their turn.\n\nShields can negate some or all of the target\'s [Reach Advantage.|Concept.ReachAdvantage] Characters who are rooted have their Reach halved. Those without an [attack of opportunity|Concept.ZoneOfControl] have no Reach.")),
+		InjuryThreshold = ::MSU.Class.BasicTooltip("Injury Threshold", ::Reforged.Mod.Tooltips.parseString("If the damage received to [Hitpoints|Concept.Hitpoints] is at least " + ::MSU.Text.colorRed(::Const.Combat.InjuryMinDamage) + " and is greater than a certain percentage of the current [Hitpoints|Concept.Hitpoints], the character receives an [injury|Concept.InjuryTemporary]. This percentage can be modified by certain [perks|Concept.Perk] or traits of both the attacker and the target.\n\nCertain [injuries|Concept.InjuryTemporary] require this percentage to be at least a certain value before they can be inflicted, with heavier [injuries|Concept.InjuryTemporary] requiring a higher percentage.\n\nFor example the threshold for [Cut Arm|Skill+cut_arm_injury] is " + ::MSU.Text.colorRed(getThresholdForInjury("injury/cut_arm_injury") + "%") + " and that of [Split Hand|Skill+split_hand_injury] is " + ::MSU.Text.colorRed(getThresholdForInjury("injury/split_hand_injury") + "%") + ".")),
```

Comment:
You change `colorNegative` into `colorRed`. This is a step back as we consider `colorRed` to be the inferior legacy version.